frontend/website/mkdocs.yml

```diff
@@ -6,7 +6,6 @@ repo_url: https://github.com/codaprotocol/coda/
 edit_uri: edit/develop/frontend/website/docs/
 nav:
   - Overview: index.md
-  - Coda Testnet: coda-testnet.md
```

Comment:
woot!  can you also make sure to update index.md and wherever else in the docs we point to coda-testnet.md, and perhaps add a link somewhere to the new landing page?